p2p/src/main/java/bisq/network/p2p/storage/P2PDataStorage.java

```diff
@@ -205,24 +214,61 @@ public GetUpdatedDataRequest buildGetUpdatedDataRequest(NodeAddress senderNodeAd
         return new GetUpdatedDataRequest(senderNodeAddress, nonce, this.getKnownPayloadHashes());
     }
 
+    /**
+     * Create the special key. <br><br>
```

Comment:
I somehow can agree on 'special key' to not be ideal. Fix coming up...

On the other hand, I fully agree on manipulating Version strings is downright bad and not desirable. However, refactoring `version` format would affect big portions of Bisq. Thus, it would directly violate our doctrine of keeping PR small and right to the point. One has to draw the line somewhere, here it is.